core/src/main/scala/chisel3/experimental/verification/package.scala

```diff
@@ -9,16 +9,10 @@ import chisel3.internal.sourceinfo.SourceInfo
 
 package object verification {
 
-  /** Named class for assertions. */
-  final class Assert(val predicate: Bool) extends BaseSim
-
-  /** Named class for assumes. */
-  final class Assume(val predicate: Bool) extends BaseSim
-
-  /** Named class for covers. */
-  final class Cover(val predicate: Bool) extends BaseSim
-
   object assert {
+    /** Named class for assertions. */
+    final class Assert(val predicate: Bool) extends BaseSim
+
```

Comment:
also moved these into their corresponding objects for consistency with [this comment](https://github.com/chipsalliance/chisel3/pull/1992/files#r662476858)